lte/gateway/c/oai/tasks/sgw/sgw_task.c

```diff
@@ -172,6 +177,7 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
       int32_t rc = sgw_handle_ip_allocation_rsp(
           spgw_state, &received_message_p->ittiMsg.ip_allocation_response,
           imsi64);
+      put_spgw_state();
```

Comment:
are we saving much by pushing this inside case statements? maybe adopt an approach similar to s1ap task with a flag set as true and if there is no need to update state set it as false in few of these cases?

Reply:
this change slipped on this PR, I am planning to do this on a different PR to take care of this similarly to the s1ap task pattern

Reply:
I will revert this change